src/views/SpaceSettings.vue

```diff
@@ -414,6 +419,12 @@ onBeforeRouteLeave(async () => {
           focus-on-mount
         >
         </BaseInput>
+        <TuneCheckbox
+          id="space-delete-acknowledge"
+          v-model="modalDeleteSpaceAcknowledge"
+          hint="I acknowledge that I can not create a new space with the same name ENS domain name"
```

Comment:
Grammar at end doesn't seem right and would be cool to include the ens domain they won't be able to use in the text

Reply:
What about now ?

Reply:
Looks good, we could also use the domain name in the warning message above

Reply:
Done